### ISSUE TITLE
Include total PR size with Average and Max

### ISSUE THREAD

**Issue by gitblit** [pullreminders/backlog#171]
Leaderboard->PR Size.

It would be nice to know the total size of contributed PRs along with the existing Avg Size, Max Size, and Count.

**Comment by abinoda** [comment]
@gitblit Thanks James! Could you elaborate more on your use case?

**Comment by gitblit** [comment]
It's hard to judge individual velocity and contributions.  I'm looking for more insight on an individual's contributions during a window relative to peers.  PR count is insufficient, although interesting.  Same with Avg and Max Size.  Total impact/churn would help to round out the stats.

**Comment by abinoda** [comment]
@gitblit I'm intentionally playing devil's advocate, but can you sell me on the value and/or real life use case for Impact/Churn (moreso the former)?

I'm aware these are metrics which exist in certain other tools but as a former engineering manager myself, I have concerns about going down the road of tying lines of code written (augmented with other factors) in ant way to the idea of measuring productivity/output/velocity of an engineer. Anecdotally I have not heard great reactions from developers either.

**Comment by gitblit** [comment]
I get that, but then why have any stats at all?  They are all tiles (good or bad) in the mosaic of productivity.  Some tiles are about code contributions, but there are many others to complete the picture.  I'm looking for more info on the code tiles.

**Comment by abinoda** [comment]
@gitblit I agree with that. But I think there's a spectrum in terms of how directly useful a metric is as well as risk of misuse and harm from distorted incentives. "impact" seems like something that could place importance on something that I think most engineers would say is not at all a measure of "Impact". What do you think?

I'm curious if there's be a way to approach this in a more binary way where you get visibility into code contributions without it being a compareitve metric, and avoiding creating distorted incentives.

I feel Impact only creates an illusion of being able to measure velocity... I'm curious to know some real-life scenarios or decisions where you would find Impact helpful.
